4-sample-snap-in/manifest.yaml

```diff
@@ -31,6 +31,13 @@ inputs:
       ui:
         display_name: Should extra comment be added?
 
+    - name: input_field_array
+      description: List of names to add as comment.
+      field_type: array
+      default_value: ["name1", "name2"]
+      ui:
+        display_name: List of extra names
```

Comment:
you will need to add a base_type as well.
PTAL: https://docs.devrev.ai/beta-api-spec#/schemas/schema-field-descriptor